### PR TITLE
Adds clustering support

### PR DESCRIPTION
**Pull request by m4rkmckenna**

### CODE REVIEW

**Comment by drigodwin** [comment]
Just noticed, this should be `centos7-software-process` not `centos-software-process`

File: catalog/rethinkdb/rethinkdb.bom [hunk 1]
```diff
@@ -105,6 +114,17 @@ brooklyn.catalog:
...
+            brooklyn.config:
+              enricher.triggerSensors:
+                - host.address
+                - rethinkdb.cluster.port
+              enricher.targetSensor: $brooklyn:sensor("rethinkdb.cluster.host.and.port")
+              enricher.targetValue:
+                $brooklyn:formatString:
+                  - "%s:%d"
+                  - $brooklyn:attributeWhenReady("host.subnet.address")
+                  - $brooklyn:config("rethinkdb.cluster.port")
```
Comment (drigodwin) [inline catalog/rethinkdb/rethinkdb.bom:126]: So this seems to no longer be inherited (AMP 4.4) which means the bp never comes up

**Review by drigodwin — CHANGES_REQUESTED** [review]
No longer works in AMP 4.4 I'm afraid

File: catalog/rethinkdb/rethinkdb.bom (hunk 1 above)
Comment (m4rkmckenna) [reply catalog/rethinkdb/rethinkdb.bom:126]: You mean a transformer can not read the config of the entity it is on? Surly thats a bug

**Pull request merged**